metrics/integration.go

```diff
@@ -115,6 +127,13 @@ func RegisterNewIntegrationPipelineRun(snapshotCreatedTime metav1.Time, pipeline
 	RegisterPipelineRunStarted(snapshotCreatedTime, pipelineRunStartTime)
 }
 
+func RegisterReleaseLatency(snapshot *applicationapiv1alpha1.Snapshot) {
+	condition := meta.FindStatusCondition(snapshot.Status.Conditions, "AppStudioTestSucceeded")
```

Comment:
1. You should probably use `gitops.AppStudioTestSucceededCondition` instead of the string here
2. I just realized that we also check for `gitops.LegacyTestsSucceededCondition` in other parts of the code.  Is that something we still need to check for or is it so out-of-date that its not a concern? @Josh-Everett it looks like you were responsible for this code at one point.

Reply:
1. calling it that way created a circular import because gitops imports metrics. Calling the string was a way around that. Am open to suggestions on how to get around that circular import problem.
2. The function in gitops checks if AppStudioTestSucceeded is true. If its not true then it checks for LegacyTests and then returns true for 'status condition' if they are found instead of the first set. So either one of those being true is assigned to statusCondition is my understanding of this function. (does not mean I am correct!!)

Reply:
1. in metrics in `RegisterReleaseLatency` just use `startTime metav1.Time` as parameter (keep endCreateRelase as time.Now()),  and define function that extracts startTime directly in snapshot controller adapter
2. you have to write new function to get data, `confition, ok := getStatusCondition(snapshot)` or smth like that, and also keep LegacyTestsSucceededCondition there